components/gitpod-protocol/src/protocol.ts

```diff
@@ -452,9 +461,9 @@ export namespace UserEnvVar {
         if (pattern.trim() === "") {
             return "Scope must not be empty.";
         }
-        const split = pattern.split("/");
-        if (split.length < 2) {
-            return "A scope must use the form 'organization/repo'.";
+        const split = splitRepositoryPattern(pattern);
+        if (split.length < MIN_PATTERN_SEGMENTS) {
+            return "A scope must use the form 'organization/repo(/...)'.";
```

Comment:
I don't think the additional `(/...)` is helpful, given we use a very concrete example. I.e. there cannot be anything after the repo name.